src/ring_theory/ideal/operations.lean

```diff
@@ -1316,7 +1316,39 @@ lemma quotient_map_comp_mk {J : ideal R} {I : ideal S} {f : R →+* S} (H : J 
   (quotient_map I f H).comp (quotient.mk J) = (quotient.mk I).comp f :=
 ring_hom.ext (λ x, by simp only [function.comp_app, ring_hom.coe_comp, ideal.quotient_map_mk])
 
-/-- `H` and `h` are kept as separate hypothesis since H is used in constructing the quotient map -/
+/-- If `f : R ≃+* S` is a ring isomorphism and `I : ideal R`, then `map f (map f.symm) = I`. -/
+lemma map_of_equiv (I : ideal R) (f : R ≃+* S) : (I.map (f : R →+* S)).map ↑f.symm = I :=
+begin
+  have h : (f.symm : S →+* R).comp (f : R →+* S) = ring_hom.id _,
+  { ext r, simp },
```

Comment:
This statement is `ring_equiv.symm_to_ring_hom_comp_to_ring_hom`

Reply:
Thank you! I was surprised that `simp` didn't do it, but it turns out that with `f.to_ring_hom` instead of `↑f` it works.